dandalandone/POC-EyCommerce
# [API Integration] - Display Cart items in Dropdown Component

## Ticket/s

- [Display Cart items in Dropdown](https://trello.com/c/HVeYueAE/108-display-cart-items-in-dropdown)

## Changes

- add API Integration for displaying Cart Items in the Header Nav Dropdown component 

## Screenshot

- Cart has items

![image](https://github.com/dandalandone/POC-EyCommerce/assets/129832182/6756866b-99b7-4d3d-877e-ee278be2d9f5)
![image](https://github.com/dandalandone/POC-EyCommerce/assets/129832182/707864b4-52d0-49bb-82ba-791c311348e4)

- Cart is empty

![image](https://github.com/dandalandone/POC-EyCommerce/assets/129832182/abdd6efd-a5d5-4b33-81e8-8b327c1d653f)
![image](https://github.com/dandalandone/POC-EyCommerce/assets/129832182/75d85864-9327-460a-80d6-f16fff03ff5d)



## How to test?

- run script `pnpm run dev`
- head to http://localhost:5173/
- check Cart dropdown in the header navbar
- add and/or remove products in cart if needed
